bitnami/postgresql-ha/Chart.yaml

```
@@ -27,4 +27,4 @@ name: postgresql-ha
 sources:
   - https://github.com/bitnami/bitnami-docker-postgresql
   - https://www.postgresql.org/
-version: 7.1.0
+version: 7.1.1
```

As it's a feature addition, we should bump the minor version:
```suggestion
version: 7.2.0
```